### ISSUE TITLE
'make launch' failed after installing SRanipal and Logitech Wheel Plugin: assertion failed

### ISSUE THREAD

**Issue by wx983** [carla-simulator/carla#6172]
build carla 0.9.13 from source

I was able to use carla before installing SRanipal and Logitech Wheel Plugin.
I've already 'make pythonAPI' again after installing above Plugins and it reports no errors seemingly from the log information. But when 'make launch', it generates an error as beblow:
![223e76dc36e400669cbb699d514532f](https://user-images.githubusercontent.com/73030394/216302326-9e862e36-3cdf-480f-a6fd-659add7b0ced.png)

**Comment by Dreyevr** [comment]
were you able to solve this error? I have the same problem.

**Comment by wx983** [comment]
> 你能解决这个错误吗？我也有同样的问题。

Yes, try install UnrealEngine 4.26 and Carla0.9.13. At the same time, you must ensure the SRanipal Plugin is 1.3.3.0 rather than 1.3.6.+. 
I reinstalled them to solve the issue.
